.github/release-drafter.yml

```diff
@@ -15,16 +15,19 @@ categories:
     label: 'deprecation'
 exclude-labels:
   - 'skip-changelog'
+category-template: '### $TITLE'
```

Comment:
Need to find a way to make the subheaders (e.g. for v0.2.1 enhancements) a proper permalink. I.e. something like https://pygmt-git-myst-changelog.gmt.vercel.app/changes.html#release-v0-2-1-enhancements instead of https://pygmt-git-myst-changelog.gmt.vercel.app/changes.html#id3

Reply:
It seems commonmark still doesn't support anchors for headers, so maybe we can do nothing here.

Reply:
Maybe do it in sphinx instead? I see https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page#comment79146992_40225340 pointing to https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#confval-autosectionlabel_prefix_document.

Reply:
> The autosectionlabel_prefix_document configuration variable can be used to make headings which appear **multiple times but in different documents** unique.

> Useful for avoiding ambiguity when the same section heading appears in different documents.

I don't think it works for us.

Reply:
@weiji14 As we have no solution for the custom anchors, do you think we should not use h3 headers, but just use **bold** headings, i.e., use `**Documentation**`, not `### Documentation`.

Reply:
Would stick with h3 headers. Even without the permalink anchors, it'll still be easier for users to navigate quickly to the subsection on the sidebar:

![Changelog subsections in navbar](https://user-images.githubusercontent.com/23487320/108779303-854a9300-75cb-11eb-8215-4068e1c3669f.png)

And people could still point to the v0.3.0 highlights using https://www.pygmt.org/v0.3.0/changes.html#highlights, and so on for v0.4.0 and beyond.